website/docs/docs/dbt-versions/release-notes/05-Aug-2023/ide-v1.2.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: "Update: Cloud IDE v1.2 with new service"
+title: "Update: Cloud IDE v1.2 includes a new service"
 description: "August 2023: Cloud IDE now uses dbt-server to provide more reliable service and dbt Core feature parity, including support for commands like `dbt list`."
 sidebar_label: "Update: Cloud IDE v1.2"
 tags: [Aug-2023, IDE]
 date: 2023-08-03
 sidebar_position: 8
 ---
 
-We're excited to announce that we replaced the backend service that powers the Cloud IDE with a more reliable server -- dbt-server. This significant update follows the rebuild of the IDE frontend last year. We're committed to improving our IDE to provide you with a better experience.
+We're excited to announce that we replaced the backend service that powers the Cloud IDE with a more reliable server -- dbt-server. Because this release contains foundational changes, dbt Core 1.6 and higher will be using Cloud IDE v1.2 going forward. This significant update follows the rebuild of the IDE frontend last year. We're committed to improving the IDE to provide you with a better experience.
```

Comment:
```suggestion
We're excited to announce that we replaced the backend service that powers the Cloud IDE with a more reliable server -- dbt-server. Because this release contains foundational changes, dbt 1.6 and higher requires IDE v1.2. This significant update follows the rebuild of the IDE frontend last year. We're committed to improving the IDE to provide you with a better experience.
```